flexmeasures/ui/templates/base.html

```diff
@@ -46,6 +46,11 @@
     {% endblock %}
 </head>
 
+<script>
+    // Define picker as a global variable
+    var picker;
```

Comment:
This place of introducing `picker` seems a bit random. 

- We can introduce it right before the `<script type="module"` gets made (line 199)
- Or we can add it in `ui.static.js.daterange-utils-js`, and import it from there (see base.html line 203) that should be cleaner.

What do you like better?